packages/shared_preferences/shared_preferences/README.md

```diff
@@ -85,6 +85,25 @@ If this is problematic for your use case, you can thumbs up
 interest in APIs that provide direct (asynchronous) access to the underlying
 preference store, and/or subscribe to it for updates.
 
+### Migration and Prefixes
```

Comment:
This combined with the doc comments on the setPrefix method seem sufficient to me. I can add a code snippet if people disagree.